### DISCUSSION TITLE
Feature Request: add option to allow multiple characters to trigger the autocomplete popup in suggestion

### DISCUSSION THREAD

**Discussion by fu-77** [ueberdosis/tiptap#4931] [Feature Requests]
### Description

I want to  use both the full-width-character (＠) and half-width character (@) to trigger the autocomplete popup in suggestion，but now I can only use one of them by the char option ([https://tiptap.dev/docs/editor/api/utilities/suggestion#char](url))。I wish there is an option to config multiple characters to trigger the autocomplete popup in suggestion。

### Use Case

I want to use multiple characters to trigger the autocomplete popup in suggestion

### Type

New feature

**Comment by mfrashad** [comment]
+1 for this feature. Would be even better if I can trigger the autocomplete programmatically as I would prefer to have the autocomplete triggered after X seconds of idle rather than keypress.
